chipsec/module_common.py

```diff
@@ -125,4 +180,4 @@ def run(self, module_argv):
 #
 # Common module command line options
 #
-OPT_MODIFY = 'modify'
+OPT_MODIFY = 'modify'
```

Comment:
missing newline

Reply:
Is it bad to have no new line at the end of the file?

Reply:
It will not cause any functional issues but is a common coding practice to help ensure the file ends with the proper CRLF/LF and for adding new functionality/concatenation of files without changing the meaning of the code.